### DISCUSSION TITLE
Capture to tag option not working?

### DISCUSSION THREAD

**Discussion by skyborne99** [chhoumann/quickadd#520] [Q&A]
Hi, I just installed the plugin and looked through the documentation + discussions, but I couldn't figure out why capture to tag doesn't seem to work for me? If I understand correctly I should type in " #tagname " under "File Name:" in the Capture options, and options for selecting files with that tag should pop up. However, no options pop up for me even when I type in a tag that appears in hundreds of my notes. (When I type a folder name, it works just fine.)

Does this Capture to Tag option only search tags in YAML frontmatter or does it also search tags in the body of notes? Because I don't use YAML frontmatter at all, so this could be the reason...

Thanks for the great plugin by the way :)

**Comment by skyborne99** [comment]
I just made a test note with a tag in the YAML frontmatter, and it's still not showing up :/

**Comment by chhoumann** [comment]
Coming back to this very late - sorry about that.

I can't reproduce this on current QuickAdd, and the tag/suggester machinery has been rewritten more than once since 2023. To answer the question you asked back then: capture-to-tag counts both kinds of tags - typing `#tagname` in the _Capture To_ field matches notes whether the tag lives in the note body or in the YAML frontmatter (including list forms like `tags: [a, b]`).

You can also combine scopes now, e.g. `folder:Goals|tag:active|exclude-tag:done`, if a plain tag picker is too broad.

Docs: https://quickadd.obsidian.guide/docs/Choices/CaptureChoice#capturing-to-tags

If you're still around and this misbehaves on the current release, please open an issue with the tag and where it appears in the note - that I can debug. Closing this one, since the original report doesn't reproduce today.
